src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -48,9 +48,8 @@
 
 <p>
     <strong>No action is required if you have already submitted.</strong>
-    <br>Any request for deadline extensions should be sent to the instructors of the course,
-    not to the TEAMMATES support team. If you need any help or clarifications regarding TEAMMATES,
-    please email us at ${supportEmail}.
+    <br><mark>Any request for deadline extensions should be sent to the instructors of the course,</mark> not to the TEAMMATES support team.
```

Comment:
It seems like the `<mark>` tag is not well supported by outlook, so it may be better for you to do the highlighting using `style` instead.
See https://www.computerhope.com/issues/ch001391.htm.